src/stylesheets/_typography.scss

```diff
@@ -18,6 +19,28 @@ $font-family-italic: "Montserrat Italic", sans-serif;
   }
 }
 
+///////////
+// To be removed after #91
+
+/* title text with the 3D cyan-red effect */
+@mixin anaglyph-title-text {
+  @include title-text;
+  text-shadow: 3px 3px var(--anaglyph-cyan), -2px -2px var(--anaglyph-red);
+}
+
+///////////
+
+/* smaller than title-text but bigger than description */
+@mixin title-text-secondary {
+  margin: 0;
+  color: var(--white);
+  font-family: $font-family-italic;
+  font-style: italic;
+  font-weight: normal;
```

Comment:
Instead of duplicating code here, can we extend from the mixin already defined? We might also be able to use our `--` notation, but I'm also not sure if that's correct again.

Reply:
the two dashes `--` is a [CSS naming convention for variables](https://www.w3schools.com/css/css3_variables.asp#main:~:text=Note%3A%20The%20variable%20name%20must%20begin%20with%20two%20dashes%20(%2D%2D)%20and%20it%20is%20case%20sensitive!). propose any changes you find to be worthwhile!